WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerViewModel.kt

```diff
@@ -47,7 +61,9 @@ class HomePagePickerViewModel @Inject constructor(
     }
 
     fun start() {
-        fetchLayouts()
+        if (!::layouts.isInitialized || layouts.isEmpty()) {
```

Comment:
I added the if statement since I didn't want the data to reload when the user goes back and forward between the two screens.
Probably the use of reflection is not correct here. I changed the check with b21ddea4047b6ac10a2779329caf59b198ce8527

Reply:
The condition makes sense to me. Good idea using the state machine as an alternative. :+1: